Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/properties/HibernateUtilFactory.java

```diff
@@ -0,0 +1,145 @@
+/**
+ *
+ */
+package gov.hhs.fha.nhinc.properties;
```

Comment:
That is where HibernateAccessor was.  The factory is now moved to another package